tests/integration/conftest.py

```diff
@@ -427,10 +448,23 @@ def runtime_ctx(ws, sql_backend, make_table, make_schema, make_udf, make_group,
     return ctx.replace(workspace_client=ws, sql_backend=sql_backend)
 
 
-class LocalAzureCliTest(WorkspaceContext):
-    def __init__(self, _ws: WorkspaceClient, env_or_skip_fixture: Callable[[str], str]):
-        super().__init__(_ws, {})
+class LocalAzureCliTest(TestRuntimeContext, WorkspaceContext):
```

Comment:
```suggestion
class LocalAzureCliTest(TestRuntimeContext):
```
extending two context factories leads to future hard-to-debug problems. extend only one of the factories - `TestRuntimeContext` in this case.